extensions/exchanges/quadriga/products.json

```diff
@@ -34,5 +33,14 @@
     "max_size": "1000000",
     "increment": "0.00001",
     "label": "ETH/BTC"
+  },
+  {
+    "id": "LTCCAD",
+    "asset": "LTC",
+    "currency": "CAD",
+    "min_size": "0.00001",
+    "max_size": "10000",
+    "increment": "0.00001",
+    "label": "LTC/CAD"
```

Comment:
Yeah. QCX only does CAD currency and no other crypto-to-crypto trading other than ETH-BTC for the moment. But they are fiarly new, so that may change with time.